Margrit/bounds change with screen

maccxs opened the pull request.

Commit:
Since I finally understood your screen resolutsion probs

* Maybe I understood a little bit more today
* Added a script which fixes the camera  viewport
* This means that we will get borders  at the edges if  screen resolution changes
* Please build and see if it works
* Placed a sprite in the lower right corner to check this

Commit:
Colliders for all camera resolutions

This should now work when starting in 16:10  and then changing to 16:9
and vice versa (both testen in unity and webgl). Please try out
Not for merging yet.

Commit:
Added further camera so that we always have a black background

holke commented:
This seems to work nicely now, good job!
I tested in in 16:9, here everythings is as before. The colliders are at the correct place.
Resizing does not change the bounds of the map.
I then tested in 16:10 and it finally works here as well! The bounds do not change when resizing.
This is how it looks now in 16:10, we get some black bars on top and on the bottom of the map, but i think this is absolutely fine.

![image](https://user-images.githubusercontent.com/7125317/80818122-f2bbac00-8bd2-11ea-8bb6-c0020a6d4872.png)

Commit:
Added code comments and used same procedure for menus

In all static scenes that do not need colliders
attach the script "UIResolution"to the main camera
In the Canvas, set the Render mode to "World Space"
And as Event Camera choose "Main Camera"
this way our menus like all other scenes will always be
scaled to fit into the current resolution wihout cutting anything off.
